app/javascript/packages/verify-flow/index.spec.tsx

```diff
@@ -1,9 +1,41 @@
 import sinon from 'sinon';
 import { render } from '@testing-library/react';
+import * as analytics from '@18f/identity-analytics';
 import userEvent from '@testing-library/user-event';
 import { VerifyFlow } from './index';
 
 describe('VerifyFlow', () => {
+  const sandbox = sinon.createSandbox();
+
+  beforeEach(() => {
+    sandbox.spy(analytics, 'trackEvent');
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it('calls trackEvent when personal key step is submitted', async () => {
+    const { getByLabelText, getAllByText } = render(
+      <VerifyFlow
+        appName="Example App"
+        initialValues={{ personalKey: '0000-0000-0000-0000' }}
+        basePath="/"
+        onComplete={() => {}}
+      />,
+    );
+
+    const continueButton = getAllByText('forms.buttons.continue')[0];
+    await userEvent.click(continueButton);
+    const input = getByLabelText('forms.personal_key.confirmation_label');
+    const submitButton = getAllByText('forms.buttons.submit.default')[1];
+    await userEvent.type(input, '0000-0000-0000-0000');
+    await userEvent.click(submitButton);
+
+    expect(analytics.trackEvent).to.have.been.calledThrice();
```

Comment:
I don't know that we'd want to expect a specific call count, since (a) VerifyFlow shouldn't know anything about the events being tracked in individual steps, and (b) it's likely going to change over time, and this would become hard to maintain.

I think it'd be enough to just ensure that `trackEvent` was called with the event name we're expecting (the assertion after this line).

```suggestion
```